package.json

```diff
@@ -727,6 +727,11 @@
                     "description": "Environment variables passed to the pipe program.",
                     "default": {}
                   },
+                  "quoteArgs": {
+                    "type": "boolean",
+                    "description": "Flag to enable or disable quotes around 'debuggerPath'",
```

Comment:
How about this for the description:
   Should arguments that contain characters that need to be quoted (ex: spaces) be quoted? Defaults to 'true'. If set, the debugger command will no longer be automatically quoted.